examples/external-data/src/mock-external-data-service/service.ts

```diff
@@ -239,6 +239,7 @@ export async function initializeExternalDataService(props: ServiceProps): Promis
 			}
 
 			// 3. Webhook exists, attempt to remove the subscriber from the webhook
+			// eslint-disable-next-line unicorn/no-negated-condition
```

Comment:
Is it not worth fixing this? Or are you aiming to reduce churn here?

Reply:
The comments are numbered 3, 3a, etc. so I didn't want to mess with the code order.